cypress/support/config/services.js

```diff
@@ -1237,67 +2377,130 @@ const genServices = {
       articles: {
-        paths: {
-          live: ['/nepali/articles/c16ljg1v008o'],
-          test: ['/nepali/articles/cl90j9m3mn6o'],
-          local: ['/nepali/articles/cl90j9m3mn6o'],
+        environments: {
+          live: {
+            paths: ['/nepali/articles/c16ljg1v008o'],
+            enabled: true,
+          },
+          test: {
+            paths: ['/nepali/articles/cl90j9m3mn6o'],
+            enabled: true,
+          },
+          local: {
+            paths: ['/nepali/articles/cl90j9m3mn6o'],
+            enabled: true,
+          },
         },
         smoke: false,
       },
       errorPage404: {
-        paths: {
-          live: [],
-          test: [],
-          local: ['/nepali/articles/c123456abcdo'],
+        environments: {
+          live: {
+            paths: [],
+            enabled: false,
+          },
+          test: {
+            paths: [],
+            enabled: false,
+          },
+          local: {
+            paths: ['/nepali/articles/c123456abcdo'],
+            enabled: true,
+          },
         },
         smoke: false,
       },
       frontPage: {
-        paths: {
-          live: ['/nepali'],
-          test: ['/nepali'],
-          local: ['/nepali'],
+        environments: {
+          live: {
+            paths: ['/nepali'],
+            enabled: true,
+          },
+          test: {
+            paths: ['/nepali'],
+            enabled: true,
+          },
+          local: {
+            paths: ['/nepali'],
+            enabled: true,
+          },
         },
         smoke: false,
       },
       liveRadio: {
-        paths: {
-          live: ['/nepali/bbc_nepali_radio/liveradio'],
-          test: ['/nepali/bbc_nepali_radio/liveradio'],
-          local: ['/nepali/bbc_nepali_radio/liveradio'],
+        environments: {
+          live: {
+            paths: ['/nepali/bbc_nepali_radio/liveradio'],
+            enabled: true,
+          },
+          test: {
+            paths: ['/nepali/bbc_nepali_radio/liveradio'],
+            enabled: true,
+          },
+          local: {
+            paths: ['/nepali/bbc_nepali_radio/liveradio'],
+            enabled: true,
+          },
         },
         smoke: false,
       },
       mediaAssetPage: {
-        paths: {
-          live: [
-            '/nepali/news-51675223', // CPS MAP
-            // '/nepali/multimedia/2013/08/130806_boudhavideo', // TC2 MAP
-          ],
-          test: [
-            '/nepali/media-23269034', // CPS MAP with audio clip
-            // '/nepali/multimedia/2016/07/160725_tc2_testmap1', // TC2 MAP
-          ],
-          local: [
-            '/nepali/media-23269034', // CPS MAP
-            // '/nepali/multimedia/2016/07/160725_tc2_testmap1', // TC2 MAP
-          ],
+        environments: {
+          live: {
+            paths: [
+              '/nepali/news-51675223', // CPS MAP with video clip
+              // '/nepali/multimedia/2013/08/130806_boudhavideo', // TC2 MAP with video clip
```

Comment:
Can these comments be removed since you have `enabled: false` on these routes?

Reply:
For some of the services where that environment is enabled, it will still need to remain commented out (there's an AMP validation bug with TC2 MAPs, so tests fail).  I'll update the ones which are disabled.